.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+
+jobs:
+  tests:
+    docker:
+      - image: circleci/python:3.7-browsers
+      - image: circleci/mysql:5.7
+        environment:
+          MYSQL_ROOT_PASSWORD: creme
```

Comment:
Required to create the test_* databases.

Reply:
Can we have both mysql and postgresql tests ?

Reply:
@mrjmad asked first :D

Reply:
yes, but step by step. 
It's already a good first step.
Next why not mariadb and postgresql yes :)